src/main/java/racingGame/Position.java

```diff
@@ -0,0 +1,36 @@
+package racingGame;
+
+public class Position {
+    private final int value;
+
+    private Position(int value) {
+        validate(value);
+        this.value = value;
+    }
```

Comment:
```suggestion
    private Position() {
        this(0);
    }

    private Position(int value) {
        validate(value);
        this.value = value;
    }
```
정적 팩터리 메서드 대신 위와 같이 생성자로 구현해도 되지 않을까?